flink-table/flink-sql-parser/src/main/codegen/includes/parserImpls.ftl

```diff
@@ -1348,6 +1352,29 @@ SqlCreate SqlCreateTable(Span s, boolean replace, boolean isTemporary) :
         String p = SqlParserUtil.parseString(token.image);
         comment = SqlLiteral.createCharString(p, getPos());
     }]
+    [
+        <DISTRIBUTED>
+        (
+            <INTO> { bucketCount = Literal(); } <BUCKETS>
+            |
+            (
+                <BY> (
+                    <HASH> { distributionKind = "HASH"; }
+                    | <RANGE> { distributionKind = "RANGE"; }
+                    | { distributionKind = null; }
```

Comment:
nit: do you think it makes sense to have an enum for the distributionKind?

Reply:
Yeah, that could make sense.  

Are there any other obvious enums in the SQL Parser that I can/should crib off of?